modules/cnf-about-ptp-and-clock-synchronization.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-about-ptp-and-clock-synchronization_{context}"]
+= About PTP and clock synchronization error events
+
+Cloud native applications such as virtual RAN require access to notifications about hardware timing events that are critical to the functioning of the overall network. Fast event notifications are early warning signals about impending and real-time PTP clock synchronization events. PTP clock synchronization errors can negatively affect the performance and reliability of your low latency application, for example, a vRAN application running in a distributed unit (DU).
+
+Loss of PTP synchronization is a critical error for a RAN network. If synchronization is lost on a node, the radio may be shut down and the network OTA traffic may be shifted to another node in the wireless network. Fast event notifications mitigates against these workload errors by allowing cluster nodes to communicate PTP clock sync status to the vRAN application running in the DU.
```

Comment:
1. OTA acronym undefined. Consider spelling out `Over the Air` when it first appears.

2. Third sentence is verbose. Consider revising. 

**Sample rewrite:**
Fast event notifications mitigate workload errors by allowing cluster nodes to communicate PTP clock sync status to the vRAN application running in the DU.